services/data_plane_manager/src/main/java/com/futurewei/alcor/dataplane/client/grpc/DataPlaneClientImplV2.java

```diff
@@ -306,8 +308,11 @@ public void onCompleted() {
                 finishLatch.countDown();
             }
         };
-
         StreamObserver<Goalstate.GoalStateV2> requestObserver = asyncStub.pushGoalStatesStream(responseObserver);
+        if (arionGatwayEnabled) {
+            goalStateV2 = goalStateV2.toBuilder().clearNeighborStates().build();
+        }
```

Comment:
@DavidLiu506 What does the clearNeighborStates do here?

Reply:
If ArionGateway enable, neighbor state do not send to ACA to create a neighbor rule because package will route to gate way.